app/controllers/api/account.php

```diff
@@ -2113,6 +2114,7 @@
 
 
         $mails
+            ->setSmtp($project->getAttribute('smtp', []))
```

Comment:
Same suggestions about the complex structure.

Reply:
updated